src/umpire/util/io.cpp

```diff
@@ -19,6 +19,10 @@
 #include "umpire/util/Macros.hpp"
 #include "umpire/util/OutputBuffer.hpp"
 
+#if defined(_MSC_VER)
+#define UMPIRE_ENABLE_FILESYSTEM
```

Comment:
I'm not sure I understand this.  Is `UMPIRE_ENABLE_FILESYSTEM` one of the macros controlled via `cmake`?  If so, shouldn't this check be done there?

Reply:
Yes, it should. For some reason the check is not working and I think @kab163 just added this to test some things out.

Reply:
That is a good question. When looking at this issue, the check here [https://github.com/LLNL/Umpire/blob/6068966b966de1560cb3783e47d6d6fcbc70cd62/cmake/SetupCompilerFlags.cmake#L14](url) is supposed to turn that flag on when needed. However, it suddenly stopped working for some reason. I tried to test that check on the dugout, but it would pass fine (I could get this check to work fine if cmake 3.19 was used and the policy set - see change in CMakeLists). That wouldn't work with the Windows CI. So, until that is sorted out, this forces that flag for Windows and the test can continue and hopefully pass. It's a temporary thing until I can figure out a better solution.

Reply:
It's probably better explained here (https://rzlc.llnl.gov/jira/browse/UM-800)